source/extensions/filters/network/mysql_proxy/mysql_codec_clogin.cc

```diff
@@ -155,6 +162,49 @@ DecodeStatus ClientLogin::parseResponse41(Buffer::Instance& buffer) {
     ENVOY_LOG(debug, "error when parsing auth plugin name of client login message");
     return DecodeStatus::Failure;
   }
+  if (client_cap_ & CLIENT_CONNECT_ATTRS) {
+    // length of all key value pairs
+    uint64_t kvs_len;
+    if (BufferHelper::readLengthEncodedInteger(buffer, kvs_len) != DecodeStatus::Success) {
+      ENVOY_LOG(debug, "error when parsing length of all key-values in connection attributes of "
```

Comment:
Rephrase to 
when parsing _total_ length of key values ...

Reply:
fixed.